db/models/index.js

```diff
@@ -1,7 +1,19 @@
 const Product = require("./Product");
 const Category = require("./Category");
+const Cart = require("./Cart");
+const LineItem = require("./LineItem");
+const User = require("./User");
 
 Product.belongsTo(Category);
 Category.hasMany(Product);
 
-module.exports = { Product, Category };
+LineItem.belongsTo(Product);
+Product.hasMany(LineItem);
+
+LineItem.belongsTo(Cart);
+Cart.hasMany(LineItem);
+
+Cart.belongsTo(User);
+User.hasMany(Cart);
```

Comment:
Should one user only have one cart? The cart can hold many products and lineItems?

Reply:
Good catch. I changed that relationship.